backend/services.plr-intake.tests/Features/Records/Search.cs

```diff
@@ -49,4 +49,86 @@ public async void SearchRecords_SingleRecord_SingleCpn()
         Assert.Single(result);
         Assert.Equal(record.Cpn, result.Single());
     }
+
+    [Theory]
+    [MemberData(nameof(IdMatchTestData))]
+    public async void SearchRecords_TrimsTo5DigitsAndLeadingZeros_Match(string dbId, string searchId)
```

Comment:
Would it be better to refactor the two Theory methods so they share the code from line 57-77?

Reply:
Quick note: I have simplified the tests a little bit since this comment, so there are less lines of code duplicated now.

I understand the desire to DRY out the code, but I don't think it is worth it here. Tests should ideally be verbose and simple to make them easy to reason about. There isn't really a nice simple way of refactoring out the duplicated lines since there are at least two objects that are being returned (either `record`, `query`, and `handler` or `record` and `result`). I also like to follow the the 
```
Arrange
Act
Assert
```
pattern as much as possible so I don't really like bundling the Arrange and Act portion together. I personally would rather repeat these lines of code to make the tests simpler and easier to reason about. I think there is a break point where this refactoring is worth it but I don't think it's here.